subsys/bluetooth/controller/Kconfig.ll_sw_split

```diff
@@ -615,7 +615,7 @@ config BT_CTLR_LLCP_COMMON_TX_CTRL_BUF_NUM
 
 config BT_CTLR_LLCP_LOCAL_PROC_CTX_BUF_NUM
 	int "Number of local control procedure contexts to be available across all connections"
-	default 4 if (BT_AUTO_PHY_UPDATE=y || BT_AUTO_DATA_LEN_UPDATE=y) && BT_CTLR_LLCP_CONN < 4
+	default 6 if (BT_AUTO_PHY_UPDATE=y || BT_AUTO_DATA_LEN_UPDATE=y) && BT_CTLR_LLCP_CONN < 4
 	default 2 if BT_CTLR_LLCP_CONN = 1
 	default BT_CTLR_LLCP_CONN if BT_CTLR_LLCP_CONN > 1
```

Comment:
I do not understand why for 2 connections only 2 context is sufficient but for 1 connection too 2 context is sufficient?

Reply:
Eventually it will depend on the application, but a reasonable number is 1 context per connection
The exception is if only one connection is allowed, then the minimum number of contexts we need is 2, see https://github.com/zephyrproject-rtos/zephyr/commit/3d9784a79e6fc40850077e07f89b525cfc836006